### ISSUE TITLE
Activating/deactivating airplane mode takes 7 seconds

### ISSUE THREAD

**Issue by userj** [ubports/ubuntu-touch#871]
### Illustrations

- Device: FP2
- Channel: 16.04/devel
- Build: 2018-09-06

### Steps to reproduce

1. access network quick settings from the pull down menu
2. toggle airplane mode
3. wait

### Expected behavior

- airplane mode toggles network connections almost immediately

### Actual behavior

- It takes about 7 seconds for the mode to be activated/deactivated.

**Comment by vlladdrakk** [comment]
I tried to reproduce this on my Nexus 4 (Mako) with build 2018-09-07 and I didn't experience such a long wait for airplane mode to work. Everything worked as normal, after turning airplane mode off it took a few seconds to connect to wifi and that is typical.

**Comment by userj** [comment]
Likewise, on most devices it's fast enough. I think it's just a Fairphone 2 thing.

**Comment by vanyasem** [comment]
Doesn't look like an issue for me. Especially if the device has 2 sim cards. I get around 5 seconds on activating/deactivating airplane mode on Samsung Galaxy S8+ with Android. It just takes time.

Could someone confirm that the same behavior doesn't happen on Android on FP2?

**Comment by luksus42** [comment]
On Fairphone Open OS its just like on/off without any big delay, 1sec maximum.
Honestly I have the impression that a lot of hardware communication is delayed on the FP2 (16.04/stable), also changing display brightness, activating camera, taking photos for example.

**Comment by thebosbird** [comment]
BQ E4.5 same behaviour. Even longer, 10 seconds. It's on 16.04
OPO is much faster, but that runs on 15.04, 2-3 seconds

**Comment by kwah-gh** [comment]
Yes it takes a while. Seconds. Next to it, for me activate/deactivate flight mode results in all connections down:
* SIM-cards getting locked so that I need to unlock them again. After unlocking SIM cards via controls appearing in swipe down menu, connections are back
   * But on the settings screen (where SIM cards are configured) notification is shown that SIM cards are not inserted and all controls are disabled (fixed only after reboot) . Although in the top bar shows that connections are active.
* Sometimes WiFi does not get up - need to toggle WiFi switch explicitly

FP2, rc-channel, 2019-W24
